recipes/pytest-mock/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "pytest-mock" %}
+{% set version = "1.1" %}
+{% set hash_type = "md5" %}
+{% set hash = "671c00dbbffee75d197eeec38c9bee91" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.python.org/packages/99/0e/45906c1e876b175cb51d8710075be900948f44a5f6a92c90095bdcd846c8/pytest-mock-{{ version }}.zip
```

Comment:
Use `https://pypi.io/packages/source/p/pytest-mock/pytest-mock-{{ version }}.zip` instead of the URL above.